### PR TITLE
Update to krona. Now uses krona 2.6 from github

### PR DESCRIPTION
**Pull request by nekrut**
Removed now defunct `-a` option

### CODE REVIEW

File: packages/package_krona_2_6/tool_dependencies.xml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0"?>
+<tool_dependency>
+    <package name="perl" version="5.18.1">
+        <repository name="package_perl_5_18" owner="iuc" prior_installation_required="True" />
+    </package>
+    <package name="krona" version="2.6">
+        <install version="1.0">
+            <actions>
+                <action type="download_by_url">https://github.com/marbl/Krona/releases/download/v2.6/KronaTools-2.6.tar</action>
```
Comment (bgruening) [inline packages/package_krona_2_6/tool_dependencies.xml]: I'm fine with this @nekrut I just wanted to point to this kind of perl packages:

https://github.com/galaxyproject/tools-iuc/blob/master/packages/package_bioperl_1_6/tool_dependencies.xml

Not sure if this works with github package, but these installations are working with CRAN packages.

Comment (bgruening) [inline packages/package_krona_2_6/tool_dependencies.xml]: You should also add a `sha256sum=""` to your download.

**Comment by bgruening** [comment]
LGTM!

**Comment by nsoranzo** [comment]
Thanks @nekrut!

**Pull request merged**